client/lib/plans/constants.js

```diff
@@ -631,6 +649,26 @@ export const FEATURES_LIST = {
 		getTitle: () => 'Unlimited Backup Space', //PLANS A/B TEST: Translate if test passes
 	},
 
+	[ FEATURE_FREE_WORDPRESS_THEMES ]: {
+		getSlug: () => FEATURE_FREE_WORDPRESS_THEMES,
```

Comment:
It is my understanding that we pay for these translations. I don't want to waste the money if the test doesn't pass. I'm happy to go wrap the translate function around strings that have been translated already but would prefer not sending the new ones over just yet.